app/assets/stylesheets/layout/page_specific/_content_hub.scss

```diff
@@ -0,0 +1,234 @@
+// Content Hub Variables --
+// Colours
+$content-hub-title-color: $color-black;
+$content-hub-bgcolor: $color-grey-eight;
+$content-hub-module-bgcolor: $color-white;
+$content-hub-module-reversecolor: $color-white;
+$content-hub-module-reverseBGcolor: $color-black;
+
+// Widths
+$content-hub-page-width: gridsystem-width(12) - $gutter-width; // unit: px
+$content-hub-modules-gutter: $baseline-unit * $base-font-size * 1px / ($baseline-unit * 0 + 1); // unit: px
+$content-hub-horizontal-space: $content-hub-modules-gutter * 3; // unit: px
+$content-hub-width-one-module: px((($content-hub-page-width) - ($content-hub-modules-gutter * 2)) / 3px); // units: em
+$content-hub-width-two-module: ($content-hub-width-one-module * 2) + px($content-hub-modules-gutter / ($content-hub-modules-gutter * 0 + 1)); // units: em
+$content-hub_width_three-module: px($content-hub-page-width / ($content-hub-page-width * 0 + 1)); // units: em
+$content-hub-module-margin-width: (px(($content-hub-horizontal-space) / ($content-hub-horizontal-space * 0 + 1)) / $content-hub-width-one-module) * 100%; // unit: %
```

Comment:
This is getting complex... Maybe it's time for a new grid system that's more flexible for gutter-less grids! OK for this page though I think.